bedrock/firefox/templates/firefox/products/index.html

```diff
@@ -157,8 +149,6 @@ <h3 class="mzp-c-menu-list-title">{{ ftl('firefox-products-get-pocket') }}</h3>
       {% include 'includes/structured-data/software/firefox-monitor-software.json' %}
       ,
       {% include 'includes/structured-data/software/firefox-lockwise-software.json' %}
-      ,
-      {% include 'includes/structured-data/software/firefox-send-software.json' %}
```

Comment:
I see a couple more templates using this file still: https://github.com/mozilla/bedrock/search?q=%7B%25+include+%27includes%2Fstructured-data%2Fsoftware%2Ffirefox-send-software.json%27+%25%7D&unscoped_q=%7B%25+include+%27includes%2Fstructured-data%2Fsoftware%2Ffirefox-send-software.json%27+%25%7D